daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/types/Ledger.scala

```diff
@@ -696,7 +696,13 @@ object Ledger {
         authParties =>
           this.authorize(
             nodeId = nodeId,
-            passIf = stakeholders.intersect(authParties).nonEmpty,
+            passIf = fetch.actingParties match {
+              case Some(actors) => actors subsetOf authParties
```

Comment:
This part worries me most with this change.

Reply:
Please check additionally that the `actors` are not empty.

Reply:
done.